src/test/java/org/broadinstitute/consent/http/db/DacDAOTest.java

```diff
@@ -258,7 +270,7 @@ void testFindMembersByDacId() {
     List<User> dacMembers = dacDAO.findMembersByDacId(dac.getDacId());
     assertNotNull(dacMembers);
     assertFalse(dacMembers.isEmpty());
-    assertEquals(dacMembers.size(), 4);
+    assertEquals(4, dacMembers.size());
```

Comment:
I know this test doesn't use hamcrest, but if it did, you could replace
```
    assertNotNull(dacMembers);
    assertFalse(dacMembers.isEmpty());
    assertEquals(4, dacMembers.size());
```

with
```
    assertThat(dacMembers, hasSize(4));
```

Reply:
I'm revamping this pretty heavily anyway ... might as well go all the way.